hijack/contrib/admin/apps.py

```diff
@@ -0,0 +1,23 @@
+from django.apps import AppConfig
+from django.contrib.auth import get_user_model
+
+
+class HijackAdminConfig(AppConfig):
+    name = "hijack.contrib.admin"
+    label = "hijack admin"
```

Comment:
https://docs.djangoproject.com/en/3.1/ref/applications/#django.apps.AppConfig.label

> This attribute allows relabeling an application when two applications have conflicting labels. It defaults to the last component of name. It should be a valid Python identifier.

```suggestion
    label = "hijack_admin"
```